Mathlib/Data/Option/Defs.lean

```diff
@@ -24,27 +26,28 @@ def lift_or_get (f : α → α → α) : Option α → Option α → Option α
 | none, some b => some b
 | some a, some b => some (f a b)
 
-/-- Lifts a relation `α → β → Prop` to a relation `option α → option β → Prop` by just adding
+/-- Lifts a relation `α → β → Prop` to a relation `Option α → Option β → Prop` by just adding
 `none ~ none`. -/
 inductive rel (r : α → β → Prop) : Option α → Option β → Prop
 | /-- If `a ~ b`, then `some a ~ some b` -/
   some {a b} : r a b → rel r (some a) (some b)
 | /-- `none ~ none` -/
   none : rel r none none
 
+/-- Traverse an object of `Option α` with a function `f : α → F β` for an applicative `F`. -/
 protected def traverse.{u, v} {F : Type u → Type v} [Applicative F] {α β : Type _} (f : α → F β) :
   Option α → F (Option β)
 | none => pure none
 | some x => some <$> f x
 
-/-- If you maybe have a monadic computation in a `[monad m]` which produces a term of type `α`, then
-there is a naturally associated way to always perform a computation in `m` which maybe produces a
-result. -/
+/-- If you maybe have a monadic computation in a `[Monad m]` which produces a term of type `α`,
+then there is a naturally associated way to always perform a computation in `m` which maybe
+produces a result. -/
 def maybe.{u, v} {m : Type u → Type v} [Monad m] {α : Type u} : Option (m α) → m (Option α)
 | none => pure none
 | some fn => some <$> fn
 
-/-- Map a monadic function `f : α → m β` over an `o : option α`, maybe producing a result. -/
+/-- Map a monadic function `f : α → m β` over an `o : Option α`, maybe producing a result. -/
 def mmap.{u, v, w} {m : Type u → Type v} [Monad m] {α : Type w} {β : Type u} (f : α → m β)
     (o : Option α) : m (Option β) :=
   (o.map f).maybe
```

Comment:
This should be called `mapM`, for consistency with monadic functions in Lean 4 core.